### PR TITLE
Proofreading for cli-tooling.adoc

### PR DESCRIPTION
**Pull request by gsmet**
Also removed content related to versions that are very old.

@michalvavrik could you have a look at this PR given you were working in this area?

### CODE REVIEW

File: docs/src/main/asciidoc/cli-tooling.adoc [hunk 1]
```diff
@@ -302,7 +302,7 @@ If you don't see the output you expect, use `--help` to verify that you are invo
 [[project-creation]]
 === Creating a new project
 
-To create a new project we use the `create` command
+To create a new project, use the `create` command
```
Comment (michalvavrik) [inline docs/src/main/asciidoc/cli-tooling.adoc:305]: Right above (can't comment on lines you didn't change) is [source,shell] example right after the sentence:

> Use `--help` to display help information with all the available commands:

I'm not sure if it is not intention (for brevity), but the example is missing certain options. Also it differs from what I have output by Quarkus CLI `999-SNAPSHOT` in following lines:

**old**
`Usage: quarkus [-ehv] [--verbose] [-D=<String=String>]... [COMMAND]`
**new**
```
Usage: quarkus [-ehv] [--refresh] [--verbose] [--config=CONFIG]
               [-D=<String=String>]... [COMMAND]
```

Options are missing 
```
     --refresh         Refresh the local Quarkus extension registry cache
      --config=CONFIG   Configuration file
```

and to make it shorter there are options whose description is not matching: `--help`, `--version`, there is missing `deploy` command and `plugin`, `update`, `test`

**Review by michalvavrik — COMMENTED** [review]
thanks @gsmet, your changes lgtm, I spotted few other points, you might want to address some of them

**Comment by gsmet** [comment]
@michalvavrik thanks I will have a look (don't know when, though!)

**Comment by gsmet** [comment]
@michalvavrik I think I addressed all your comments and that we should be good to go.

**Review by michalvavrik — APPROVED** [review]
I think so @gsmet , lgtm, thank you

**Pull request merged**